src/html/login.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="css/login.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+	<title>Login</title>
+</head>
+<body>
+    
+<section id="form">
+    <header>
+        <h1>Login</h1>
+    </header>
+    <main>
+        <div id="form-tabs">
+            <div id="form-tab-signin" class="form-tab" name="chosen">Sign in</div>
+            <div id="form-tab-signup" class="form-tab">Sign up</div>
+            <div id="form-tab-line"></div>
+        </div>
+        <div id="form-sections">
+            <section id="form-signin">
+                <div id="signinForm">
+                    <div class="input">
+                        <h3>E-mail</h3>
+                        <input type="text" placeholder="E-mail" id="signin-email"/>
+                    </div>
+                    <div class="input">
+                        <h3>Password</h3>
+                        <input type="password" placeholder="Password" id="signin-password"/>

```

Comment:
Password? Strona jest po polsku czy angielsku?

Reply:
To chyba nie ten pr...

Reply:
Można założyć taska na to